PWGGA/GammaConv/macros/AddTask_GammaConvCalo_pp.C

```diff
@@ -1013,7 +1013,7 @@ void AddTask_GammaConvCalo_pp(
  //*************************************************************************************************
   // 7 TeV PHOS for omega analysis variations
   //*************************************************************************************************
-  } else if ( trainConfig == 380){ // pileup
```

Comment:
I had some weird merge conflicts... I added the line that I had removed, however, it seems that the history is now a bit weird